core/src/com/unciv/models/metadata/GameSettings.kt

```diff
@@ -13,6 +17,8 @@ class GameSettings {
     var checkForDueUnits: Boolean = true
     var singleTapMove: Boolean = false
     var language: String = "English"
+    @Transient
+    var locale: Locale? = null
```

Comment:
Loading settings from disk will need to set this too, and I can't see it. Must be blind.

Reply:
Yup, I see it.